### PR TITLE
add code to download NOAA GEFS forecasts from EFI

### PR DESCRIPTION
**Pull request by dlebauer**

### CODE REVIEW

**Comment by KristinaRiemer** [comment]
I couldn't get the `neon4cast` package to install, so I cloned the repo and sourced the noaa functions. 

Does it make the most sense to have this forecasted data be generated in its own script, or should it be included in the pre-existing `curate_weather.R` one? Also, is it easier to leave this forecasted data as its own csv, or could it be joined to the gcc data like the daymet data was?

**Comment by KristinaRiemer** [comment]
There's no date and time for each of the rows, but I think there should be? Are we supposed to generate that ourselves?

File: pheno_images/curate_weather_forecasts.R [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# devtools::install_github('eco4cast/neon4cast')
+# install.packages('readr')
+library(neon4cast)
+pheno_sites <- c("HARV", "BART", "SCBI", "STEI", "UKFS", "GRSM", "DELA", "CLBJ"
```
Comment (KristinaRiemer) [inline pheno_images/curate_weather_forecasts.R:4]:
```suggestion
pheno_sites <- c("HARV", "BART", "SCBI", "STEI", "UKFS", "GRSM", "DELA", "CLBJ)"
```

File: pheno_images/curate_weather_forecasts.R [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+)
```
Comment (KristinaRiemer) [inline pheno_images/curate_weather_forecasts.R:5]:
```suggestion

```

File: pheno_images/curate_weather_forecasts.R [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+download_noaa(pheno_sites)
+noaa_fc <- stack_noaa()
+readr::write_csv(noaa_fc, file = paste0('NOAA_GEFS_35d_', Sys.Date(), '.csv'))
```
Comment (KristinaRiemer) [inline pheno_images/curate_weather_forecasts.R:8]:
```suggestion
readr::write_csv(noaa_fc, file = paste0('pheno_images/NOAA_GEFS_35d_', Sys.Date(), '.csv'))

```

**Review by KristinaRiemer — APPROVED** [review]

**Comment by dlebauer** [comment]
> There's no date and time for each of the rows, but I think there should be? Are we supposed to generate that ourselves?

probably something like 

```
x <- noaa_fc %>% mutate(date = startDate + lubridate::hours(time))
```

> Does it make the most sense to have this forecasted data be generated in its own script, or should it be included in the pre-existing `curate_weather.R` one? 

> Also, is it easier to leave this forecasted data as its own csv, or could it be joined to the gcc data like the daymet data was?

will defer to @DebasmitaPal1206, but future weather won't have gcc .. .it would be generated like `gcc_pred <- predict(gcc_model, forecasts)`

**Comment by DebasmitaPal1206** [comment]
@KristinaRiemer  Please keep forecasted weather data in a separate .csv fie. These forecasted weather parameters should be matched with the weather parameters in gcc_weather.csv file.

**Comment by dlebauer** [comment]
follow up in https://github.com/genophenoenvo/neon-datasets/issues/37

**Pull request merged**